.travis.yml

```
@@ -0,0 +1,12 @@
+language: java
+env:
+  - TEST_DIR=yajco-example-deskNielsen
+  - TEST_DIR=yajco-example-extMathExpressions
+  - TEST_DIR=yajco-example-gettingStarted
+  - TEST_DIR=yajco-example-imperativeLang
+  - TEST_DIR=yajco-example-json
+  - TEST_DIR=yajco-example-mathExpressions
+  - TEST_DIR=yajco-example-robotKarel
+  - TEST_DIR=yajco-example-simpleRobot
+  - TEST_DIR=yajco-example-stateMachine
+script: cd $TEST_DIR && mvn verify && mvn clean
```

Is it required to call `mvn clean`? Doesn't Travis remove whole project directory after running the tests?

```suggestion
script: cd $TEST_DIR && mvn verify
```

---

No it's not required. Just a habit to clean workspace after some work.